linkerd/distribute/src/service.rs

```diff
@@ -150,6 +150,17 @@ impl<K: Clone, S: Clone> Clone for Distribute<K, S> {
     }
 }
 
+impl<K, S> Default for Distribute<K, S> {
+    /// Returns an empty distribution. This distribution will never become ready/
```

Comment:
whoops, typo on my part:
```suggestion
    /// Returns an empty distribution. This distribution will never become ready.
```